engine/runtime/src/main/java/org/enso/interpreter/runtime/state/ExecutionEnvironment.java

```diff
@@ -108,4 +92,9 @@ public static ExecutionEnvironment forName(String name) {
         throw new IllegalArgumentException("Unsupported Execution Environment `" + name + "`");
```

Comment:
Looks like there is a missing case for `dataflowStacktrace`.

Reply:
Fixed in dd25a6f256918a675898e23ca95172ab3d5d463a